types/mongoose/index.d.ts

```diff
@@ -1144,7 +1144,7 @@ declare module "mongoose" {
 
   interface MongooseDocumentOptionals {
     /** The string version of this documents _id. */
```

Comment:
What about this comment here? Since the property isn't necessarily a string anymore, perhaps we could describe it further?

Reply:
(Also because it doesn't necessarily reference the `_id`.

Reply:
Ahh good point.

Reply:
Updated https://github.com/DefinitelyTyped/DefinitelyTyped/pull/21856/commits/8e4344f381c3f4314ddf87c2aa43da6c67dea4a2